apache/datadog_checks/apache/apache.py

```diff
@@ -2,6 +2,7 @@
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
 import requests
+from urllib3.exceptions import InsecureRequestWarning
```

Comment:
Thanks. Tests are now failing because of the order of the imports. Can you change the order so it is:

```
import requests
from six.moves.urllib.parse import urlparse
from urllib3.exceptions import InsecureRequestWarning
```

And tests should then pass.